js/screens/OnBoarding/OnBoarding.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { Text, View, Image } from 'react-native';
+import Onboarding from 'react-native-onboarding-swiper';
+import styles from './styles';
+
+const SubtitleComp = props => {
+  return <Text style={styles.text}>{props.subtitle}</Text>;
+};
+
+const OnBoarding = ({ navigation }) => {
+  const Pages = [
+    {
+      title: ' ',
+      image: (
+        <Image
+          source={require('../../assets/images/Onboarding/Browse.png')}
+          style={styles.images}
+        />
+      ),
+      subtitle: (
+        <SubtitleComp
+          subtitle={`Create a listing and connect with Vancouver locals to exchange home health equipment.`}
+        />
+      ),
+      backgroundColor: '#fff',
+      alterBottomColor: '#fff',
```

Comment:
All the styles should be in the styles.js file. If you can't make a const for this since you are using it multiple times on this page.